read.g

```diff
@@ -24,6 +24,14 @@ if IsBound(IO_PkgThingsToRead) then
     Unbind(IO_PkgThingsToRead);
 fi;
 
+if IsBound( MakeThreadLocal ) then
+    Perform(
+            [
+             "IO",
+             ],
+            MakeThreadLocal );
```

Comment:
Thanks for the patch! But that's very strange formatting, against all the usual GAP conventions.

Why not just write this, though?
```gap
if IsHPCGAP then
  MakeThreadLocal("IO");
fi;
```

Reply:
I copied the code from other packages homalg-packages, where I had to make much more global variables thread-local, here I only need to do it for `IO'. Besides, I was unaware of `IsHPCGAP'. I'll use you code an push a new commit. Thanks.